hw/dv/tools/dvsim/vcs.hjson

```diff
@@ -15,6 +15,8 @@
                "-Mdir={build_ex}.csrc",
                "-o {build_ex}",
                "-f {sv_flist}",
+               // List multiple tops for the simulation. Prepend each top level with `-top`.
+               '''{eval_cmd} tops=(`echo {sim_tops}`); echo "${tops[@]/#/-top }"''',
```

Comment:
We've got two `{eval_cmd}` lines in this file that both compute the same thing. Maybe put it in a separate variable to avoid the unneeded extra subprocess?

Reply:
Duh, I didn't read this carefully enough! You're completely right.